website/examples/navigation-disabled.tsx

```diff
@@ -1,8 +1,13 @@
-import React from 'react';
-import { DayPicker } from 'react-day-picker';
+import React from "react";
+import { DayPicker } from "react-day-picker";
 
 export function Example() {
   return (
-    <DayPicker fromYear={2015} toYear={2025} disableNavigation mode="single" />
+    <DayPicker
+      fromYear={2015}
+      toYear={2025}
+      disableNavigation
+      mode="single"
```

Comment:
Here is an uncontrolled example that may no longer work as intended

Reply:
I prefer this version with no "magic": without specifying `selected` or `onSelect`, nothing happens. 

(I will remove the mode prop from this example)